api/src/controllers/stp.ts

```diff
@@ -39,7 +37,8 @@ export class STPController {
     @Query() _program: string
   ): Promise<string | Buffer> {
     const vars = request.query as ExecutionVars
-    return execute(request, _program, vars)
+
+    return execute(request, _program, vars, undefined)
```

Comment:
why there is an explicit need to pass undefined as an argument when it is already optional?